api/src/pcapi/alembic/versions/20240716T150036_5f46232cb72f_add_cancellationauthor_for_booking.py

```diff
@@ -0,0 +1,22 @@
+"""
+add cancellationAuthor for booking
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "5f46232cb72f"
+down_revision = "e0d7e16bcbaa"
+branch_labels: tuple[str] | None = None
+depends_on: list[str] | None = None
+
+
+def upgrade() -> None:
+    op.add_column("booking", sa.Column("cancellationAuthorId", sa.BigInteger(), nullable=True))
```

Comment:
```suggestion
    if not settings.IS_PROD:
        op.add_column("booking", sa.Column("cancellationAuthorId", sa.BigInteger(), nullable=True))
```

Je te l'ai deja dit plusieurs fois en commentaire et sur slack, il faut désactiver cette migration en prod, on la jouera manuellement